docs/additional-functionality/rapids-udfs.md

```diff
@@ -52,7 +52,7 @@ Other forms of Spark UDFs are not supported, such as:
 
 For supported UDFs, the RAPIDS Accelerator will detect a GPU implementation
 if the UDF class implements the
-[RapidsUDF](../../sql-plugin/src/main/java/com/nvidia/spark/RapidsUDF.java)
+[RapidsUDF](https://github.com/NVIDIA/spark-rapids/blob/branch-21.12/sql-plugin/src/main/java/com/nvidia/spark/RapidsUDF.java)
```

Comment:
If this is just for github pages, shouldn't these links all point to the gh-pages version, i.e.: under https://nvidia.github.io/spark-rapids/, rather than the github markdown from the source repository?

Reply:
@jlowe gh-pages branch does not have source code so i have to link it to the github source code for that specific released branch.

Reply:
Ah right, these are pointing to source.  If we're going to point to source from the Github pages, then this should point to the `main` branch since gh-pages should reflect what's been released (i.e.: on `main`).  This also has the benefit of not needing to be updated in most cases.  It only needs to be updated when the link changes or is removed, and we will get a merge conflict on the gh-pages branch if that occurs.

Reply:
Actually I am fine with main. Actually that is easier.
The only concern is that, before the official release say 22.02, the main branch merge may be done before that.
If we are fine with that, i can update all links to point to main branch.

Reply:
@jlowe  how about now? i changed everything to main branch.